### PR TITLE
feat: add jokes in french

### PR DESCRIPTION
**Pull request by lachhebo**

### CODE REVIEW

File: pyjokes/jokes_fr.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+"""
+Initial jokes from stackoverflow - provided under CC BY-SA 3.0
+http://stackoverflow.com/questions/234075/what-is-your-best-programmer-joke?page=4&tab=votes#tab-top
+"""
+
+neutral = [
+    "Quel Pokemon a une mitraillette ?   Ratatatatatatatatata",
+    "A quoi sert Internet Explorer ? À télécharger Google Chrome."
```
Comment (vpoulailleau) [inline pyjokes/jokes_fr.py:10]:
```suggestion
    "A quoi sert Internet Explorer ? À télécharger Google Chrome."
```

**Review by vpoulailleau — COMMENTED** [review]
LGTM except the mandatory non breakable space in french typography.

**Comment by alfreddagenais** [comment]
I have already added french joke here : https://github.com/pyjokes/pyjokes/pull/91 🤔

**Comment by alexsavio** [comment]
Could you please solve the conflicts?

**Pull request merged**